contract/models/account_move.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2016 Tecnativa - Carlos Dauden
+# Copyright 2018 ACSONE SA/NV.
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountInvoice(models.Model):
```

Comment:
To match the model name:

```suggestion
class AccountMove(models.Model):
```